testing/test_junitxml.py

```diff
@@ -75,7 +85,7 @@ def tag(self):
         return self.__node.tagName
 
     @property
-    def next_siebling(self):
+    def next_sibling(self):
```

Comment:
Is this public API?  If so we should note the change somewhere.

Reply:
Nope, it is used only on the test. 👍